components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,29 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+	static get properties() {
+		return {
+			color: { type: String },
+			displayed: {type: Boolean}
+		};
+	}
+
+	constructor() {
+		super();
+		this.color = "blue";
+		this.strokeWidth = 4;
```

Comment:
all dimensions are calculated based on the stroke width

Reply:
Assigning via property makes sense to me, with the default being `3` as I think that's what the designs use

Reply:
The strokeWidth is private so it should have a `_` to denote it.
https://blog.d2l.dev/2020/05/11/LitElement-Best-Practices-Gotchas/